### DISCUSSION TITLE
unable to download Better BibTeX latest release v6.7.98 (or any other)

### DISCUSSION THREAD

**Discussion by gaiaca** [retorquere/zotero-better-bibtex#2605] [Q&A] [answered]
I was trying to download Better BibTeX' latest release v6.7.98 on macbook air, but it says that the file (.xpi) is damaged. Same goes for earlier releases. Is there a version that works for sure? As the latest releases are quite recent, maybe they are incomplete yet or not comaptible? Or might it be an error of the mac?

Would be grateful for any help! 
PS: I am a coding noob, so I am for sure missing things...

**Comment by retorquere** [comment, marked as answer]
You're downloading using Firefox? Right click, save, then install from the zotero addons screen.

**Reply by gaiaca** [reply]
Yes, I used firefox. Worked perfectly now, thanks!
